src/core/automation.py

```
@@ -327,6 +331,61 @@ def request_accessibility_permissions(self) -> None:
+    def get_kindle_window_info(self) -> Optional[dict]:
+        """Kindleアプリのウィンドウ情報を取得
+        
+        Returns:
+            dict: ウィンドウ情報 (bounds, title, owner_name, window_number) または None
+        """
+        if not PYOBJC_AVAILABLE:
+            logger.warning("PyObjC not available - cannot detect window")
+            return None
+        
+        try:
+            from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)